thirdeye/thirdeye-pinot/src/test/resources/schema/create-schema.sql

```diff
@@ -268,4 +268,19 @@ create index detection_status_sdf_idx on detection_status_index(date_to_check_in
 create index detection_status_run_idx on detection_status_index(detection_run);
 create index detection_status_function_idx on detection_status_index(function_id);
 
+create table if not exists function_autotune_config_index (
```

Comment:
When you design the table, it might be good if we can use it for more generic use case (not limited to function configure) can be other configures. What I am thinking of here is: if we separate filter from anomaly detection function, can we leverage this table design.